### ISSUE TITLE
Check if admin/book.texinfo is needed

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1857]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)| @puzzlement reported|
|-|-|
|Trac ID|trac#1857|
|Type|enhancement|
|Created|2006-06-24 23:46:24Z|

There is a file in the repository admin/book.texinfo. It was last updated in 2003 by moshez. It does not seem to be called by any scripts, and I've rearranged the docs since then without updating it.

Is it used? If so, update it. Otherwise, remove it.

<details><summary>Searchable metadata</summary>

```
trac-id__1857 1857
type__enhancement enhancement
reporter__hypatia hypatia
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1151192784000000 1151192784000000
changetime__1215937737000000 1215937737000000
version__None None
owner__ 
cc__radix cc__moshez cc__exarkun cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set status to `closed`|
|-|-|

Fixed with r22169 ([#2925](https://github.com/twisted/twisted/issues/2925)).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
